_includes/footer.html

```diff
@@ -0,0 +1,49 @@
+<footer class="site-footer">
```

Comment:
Yes it is, normally the file lives in the gem for this template, but because I need to change this file I am getting it out of the gem and into the repo to be able to change it to add this link to the repo.